src/components/WorkflowForms/RedoEstimate/RedoEstimate.jsx

```
@@ -8,16 +8,17 @@ import {Link, useNavigate} from "react-router-dom";
 import Toast from "../../../components/Toast/Toast";
 import * as Yup from "yup";
 import api from "../../../utilities/api.js";
-import jb_utils from "../../../utilities/functions.jsx";
+import mva_utils from "../../../utilities/functions.jsx";
 
 import strings from "./i18n-strings";
 
 export default function RedoEstimate({
-    jobId = ""
+    caseId = "",
+    setSelectedStep
```

ESLint

[error] 'setSelectedStep' is defined but never used.

(no-unused-vars)